Make the __module__ name of HOO to be always "torch.ops.higher_order"

zhxchen17 opened the pull request:
Summary: It seems that we can just make this the default so that in the future all the ops printed in the graph should be like torch.ops.higher_order

Test Plan: CI

Differential Revision: D60530900


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

ydwu4 reviewed (approved):
Thanks!

SherlockNoMad commented:
can you make it raise an error if the naming doesn't follow convention during HOP authoring?

zhxchen17 committed:
Make the __module__ name of HOO to be always "torch.ops.higher_order" (#132775)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/132775

It seems that we can just make this the default so that in the future all the ops printed in the graph should be like torch.ops.higher_order

Test Plan: CI

Differential Revision: D60530900
